packages/neuron-ui/src/states/stateProvider/actionCreators/wallets.ts

```diff
@@ -43,6 +42,9 @@ export const updateWalletList = () => (dispatch: StateDispatch) => {
     if (isSuccessResponse(res)) {
       const payload = res.result || []
       dispatch({ type: NeuronWalletActions.UpdateWalletList, payload })
+      if (walletsCache.load().length < payload.length) {
```

Comment:
It's better to move this logic to https://github.com/nervosnetwork/neuron/blob/develop/packages/neuron-ui/src/utils/hooks/index.ts#L489-L497.

Reply:
Because the message is also displayed when a new wallet is added, it can't just be judged in `useFirstLoadApp`.

Reply:
May be we can rename `useFirstLoadApp` to `useFirstLoadWallet`